app/presenters/migration/index.rb

```diff
@@ -5,16 +5,55 @@ class Index < ApplicationPresenter
     def content
       mustache[:content] ||= begin
         {
-          title: page_content_heading
+          title: page_content_heading,
+          hero: {
+            url:              '/images/channel_hero_migrations.jpg',
+            title:            t('hero.title'),
+            subtitle:         t('hero.subtitle'),
+            attribution_text: t('hero.attribution_text'),
+            attribution_url:  'http://www.europeana.eu/portal/record/2048211/449.html',
+            license_public:   true
+          },
+
+          description: t('description'),
+          closing_remark: t('closing_remark'),
+
+          begin_link: {
+            url: new_migration_path,
+            text: t('begin_link.text'),
+            text_long: t('begin_link.text_long')
+          },
+          call_to_action: t('call_to_action'),
+          previews: [
+            {
+              caption: t('preview_1.caption'),
+              img_url: '/images/ugc-preview-1.jpg',
```

Comment:
Are these images supposed to be present already? If so, served from where?

Reply:
I think the eventual plan is to have them controlled via a CMS - for now they're coming from the assets pipeline.

Reply:
So where are they?